scripts/regenerate.sh

```diff
@@ -19,12 +19,15 @@ WORKDIR=$(mktemp -d)
 
 function finish {
   rm -rf "$WORKDIR"
+  # Revert back the PATH to client's original value
+  export PATH=RESTORED_PATH
```

Comment:
```suggestion
  export PATH=$RESTORED_PATH
```

Reply:
Also, do you mind providing some evidence of this actually working?

Reply:
Sure, please find the log [here](https://github.com/grpc/grpc-go/actions/runs/8935136639/job/24543126040?pr=7064#step:4:137) in github CI. But i feel its not good to add the PATH value in the log. WDYT?